builtins/amp-img.js

```diff
@@ -208,10 +208,14 @@ export class AmpImg extends BaseElement {
    * @private
    */
   guaranteeSrcForSrcsetUnsupportedBrowsers_() {
+    // The <img> tag does not have a src and does not support srcset
     if (!this.img_.hasAttribute('src') && 'srcset' in this.img_ == false) {
       const srcset = this.element.getAttribute('srcset');
-
-      const srcseturl = /\S+/.match(srcset);
+      const matches = /\S+/.exec(srcset);
+      if (matches == null) {
+        return;
+      }
+      const srcseturl = matches[0];
       if (srcseturl) {
         this.img_.setAttribute('src', srcseturl[0]);
```

Comment:
This sets src to a char.

Reply:
0.0 oh god. Good catch! Fixed and testing that now.

Reply:
Tested this on IE. It works.